modules/workers/clusternetworks.tf

```diff
@@ -33,11 +33,7 @@ resource "oci_core_cluster_network" "workers" {
   }
 
   lifecycle {
-    ignore_changes = [
-      display_name, defined_tags, freeform_tags,
-      instance_pools["display_name"], instance_pools["defined_tags"], instance_pools["freeform_tags"],
-      placement_configuration["availability_domain"],
-    ]
+    ignore_changes = [display_name, defined_tags, freeform_tags]
```

Comment:
I thought you had to put the tree in?

Reply:
Ignoring any key here has the effect of ignoring the whole block, unfortunately - tested a number of ways.
It doesn't look like automatic tags are being added at this level in practice.